packages/plugin-renderer-react/src/phenomicRendererReactClient.re

```diff
@@ -1,16 +1,14 @@
 [@bs.module "@phenomic/plugin-renderer-react/lib/client"]
-external createContainer_:
-  (ReasonReact.reactClass, Js.t({..})) => ReasonReact.reactClass =
+external createContainer_: (React.component('a), Js.t({..})) => React.element =
```

Comment:
I think it's more returning the same React.component (kind of), & same below. Right?

Reply:
What do you mean here?

Reply:
Should this be `(React.component('a), Js.t({..})) => React.component('a)` ? an hoc isn't returning a rendered element, but almost the same component